### PR TITLE
Develop layout_html-form

### PR DESCRIPTION
**Pull request by sergey-lipikhin**
- [DEMO LINK](https://sergey-lipikhin.github.io/layout_html-form/)
- [TEST REPORT LINK](https://sergey-lipikhin.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,131 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="test-form"
+    >
```
Comment (OleksiiBlezniuk) [inline src/index.html]:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      name="test-form"
    >
```

Comment (OleksiiBlezniuk) [reply src/index.html]: Screenshot 2023-01-23 at 18 01 17

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,131 @@
...
+        </label>
+      </fieldset>
+
+
+      <fieldset name="interestingInfo" class="form-fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?
+          <label>
+            <input type="radio" name="cats">
```
Comment (OleksiiBlezniuk) [inline src/index.html]: But what about this buttons value? How will it be displayed in forms data that is sent to server?

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,131 @@
...
+        <legend>Additional info:</legend>
+        <label class="form-field">
+          Comments:
+          <textarea name="comments" autocomplete="off"></textarea>
+        </label>
+        <label class="form-field">
+          Would you recommend us?
+          <select name="recommend">
+            <option value="bmw">yes</option>
+            <option value="audi">no</option>
```
Comment (OleksiiBlezniuk) [inline src/index.html]: This values are wrong

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (sergey-lipikhin) [reply src/index.html]: It's hard to say, I have no set up server to check the result of this request. Probably there will be empty value, for 'cats' variable.
`"cats": ""`
But I can definitely say that I should insert value attribute with different values for different radios.

Can I not respond to your remarks if I understand them? Or it is required?:)

Comment (OleksiiBlezniuk) [reply src/index.html]: You may not responding if you agree with comment and understand reason :)
Sometimes questions can specify issue in more clear way, so we using it

**Review by OleksiiBlezniuk — APPROVED** [review]
GJ!

**Pull request closed without merging**